subsys/bluetooth/controller/Kconfig

```diff
@@ -16,26 +16,27 @@ if BT_CTLR
 
 choice BT_LL_CHOICE
 	prompt "Bluetooth Link Layer Selection"
-	default BT_LL_SW
```

Comment:
Why did you remove this line?

Reply:
BT_LL_SW_SPLIT being the first item in the choice block, it is the de facto default